test/integration/patients/patient/patient-flow.js

```diff
@@ -686,79 +660,76 @@ context('patient flow page', function() {
-          action.id = `${ index + 1 }`;
+        fx.data = mergeJsonApi(testFlow, {
+          relationships: {
+            state: getRelationship(stateInProgress),
+            owner: getRelationship(teamNurse),
+          },
         });
 
-        fx.data.relationships.actions.data = flowActions;
-        fx.data.relationships.state.data.id = '33333';
-        fx.data.relationships.owner.data = {
-          id: '22222',
-          type: 'teams',
-        };
-
         return fx;
       })
       .routePatientByFlow()
       .routeFlowActions(fx => {
-        fx.data = _.first(fx.data, 4);
-
-        _.each(fx.data, (action, index) => {
-          action.id = `${ index + 1 }`;
-        });
-
-        fx.data[0].attributes.sequence = 1;
-        fx.data[1].attributes.sequence = 2;
-        fx.data[2].attributes.sequence = 3;
-        fx.data[3].attributes.sequence = 4;
-
-        fx.data[0].relationships.state.data.id = '22222';
-        fx.data[1].relationships.state.data.id = '22222';
-        fx.data[2].relationships.state.data.id = '22222';
-        fx.data[3].relationships.state.data.id = '55555';
-
-        fx.data[0].relationships.owner.data = {
-          id: '22222',
-          type: 'teams',
-        };
-
-        fx.data[1].relationships.owner.data = {
-          id: '22222',
-          type: 'clinicians',
-        };
-
-        fx.data[2].relationships.owner.data = {
-          id: '11111',
-          type: 'teams',
-        };
-
-        fx.data[3].relationships.owner.data = {
-          id: '22222',
-          type: 'teams',
-        };
+        fx.data = [
+          getAction({
+            attributes: {
+              sequence: 1,
+            },
+            relationships: {
+              state: getRelationship(stateTodo),
+              owner: getRelationship(teamNurse),
+              flow: getRelationship(testFlow),
+            },
+          }),
+          getAction({
+            attributes: {
+              sequence: 2,
+            },
+            relationships: {
+              state: getRelationship(stateTodo),
+              owner: getRelationship('22222', 'clinicians'),
+              flow: getRelationship(testFlow),
+            },
+          }),
+          getAction({
+            attributes: {
+              sequence: 3,
+            },
+            relationships: {
+              state: getRelationship(stateTodo),
+              owner: getRelationship(teamCoordinator),
+              flow: getRelationship(testFlow),
+            },
+          }),
+          getAction({
+            attributes: {
+              sequence: 4,
+            },
+            relationships: {
+              state: getRelationship(stateDone),
+              owner: getRelationship(teamNurse),
+              flow: getRelationship(testFlow),
+            },
+          }),
+        ];
 
-        fx.included = _.reject(fx.included, { type: 'flows' });
-        fx.included.push({
+        fx.included.push(getClinician({
```

Comment:
similarly it's odd that this is being pushed to the included.  we should be able to set this on the bootstrapped `routeWorkspaceClinicians` like we would elsewhere

Reply:
Updated in https://github.com/RoundingWell/care-ops-frontend/pull/1249/commits/547129d020b12f89b46152ff55d9bde5fcc8e573.